test/jdk/javax/swing/JComponent/7154030/bug7154030.java

```diff
@@ -107,7 +107,7 @@ public void run() {
             locx = bounds.x + insets.left;
             locy = bounds.y + insets.top;
             frw = bounds.width - insets.left - insets.right;
-            frh = bounds.height - insets.top - insets.bottom;
+            frh = bounds.height - insets.top - insets.bottom - 25;
```

Comment:
Will making frame undecorated help instead of this?
Can you also please give a CI job running this test on all platforms for several iterations?
Since this test has undergone multiple iterations, can you please confirm that this test still fails without 7154030 fix?

Reply:
> Will making frame undecorated help instead of this?

Yes, seems like making it undecorated fixes the issue. I can not provide many failure results since failures were pretty uncommon in the first place. As for the test run i will add link to it into the bug comment.